master/buildbot/test/unit/www/test_ldapuserinfo.py

```diff
@@ -91,7 +89,8 @@ def makeUserInfoProvider(self):
         self.userInfoProvider = ldapuserinfo.LdapUserInfo(
             uri="ldap://uri", bindUser="user", bindPw="pass",
             accountBase="accbase", groupBase="groupbase",
-            accountPattern="accpattern", groupMemberPattern="groupMemberPattern",
+            accountPattern="accpattern",
```

Comment:
I think this wasn't needed for the fix you're doing was it? Buildbot uses 100 column line length limit so the current code was fine. Could you revert this change here and in a couple of places below?

Reply:
Sorry, I didn't know. Fixed.